crates/trigger/src/lib.rs

```diff
@@ -73,6 +73,7 @@ where
         label: app.info.name,
         log_dir,
         config_resolver: app.config_resolver,
+        ..Default::default()
```

Comment:
That's fair! If any new `ExecutionContextConfiguration` items are of type `Option` or something like that `impl Default` by, well, _default_, this could make for an annoying error to catch. For now, I'll fix this my just calling `ModuleIoRedirectTypes`' `default` directly.

Reply:
done ✅